UtopiaCity/Controllers/Airport/FlightController.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using UtopiaCity.Data;
+using UtopiaCity.Models.Airport;
+using UtopiaCity.Services.Airport;
+
+namespace UtopiaCity.Controllers.Airport
+{
+    public class FlightController:Controller
+    {
+        private FlightService _flightService;
+        public FlightController(FlightService flightService)
+        {
+            _flightService = flightService;
+        }
+        public IActionResult Index()
```

Comment:
Empty lines between methods will make code look better. Same thing throughout whole file/files

Reply:
OK will fix it